ui-components/src/locales/ar.json

```diff
@@ -1410,13 +1410,13 @@
         "age": {
             "prompt": "كم عمرك؟",
             "description": "تقبل بعض العقارات الأفراد الذين تزيد أعمارهم عن 62 عامًا فقط.",
-            "label": "سنة",
-            "error": "الرجاء إدخال عمر صالح."
+            "lessThan55": "55 >",
+            "55to61": "55 - 61",
```

Comment:
nit: I bet you looked this up, but it's interesting that it's "55 - 61" and not "61 - 55" :)

Reply:
I'm glad to see you're paying attention! It actually should be 61-55, but I think our tailwind RTL plugin is too smart and flips it around again, because when I put that there it shows up incorrectly on the actual page.